### DISCUSSION TITLE
v1.1.2

### DISCUSSION THREAD

**Discussion by mxcl** [pkgxdev/brewkit#224] [Announcements]
## What's Changed
* fix git packages by @jhheider in https://github.com/pkgxdev/brewkit/pull/222

**Full Changelog**: https://github.com/pkgxdev/brewkit/compare/v1.1.1...v1.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/pkgxdev/brewkit/releases/tag/v1.1.2'>v1.1.2</a>.</em>

**Comment by jhheider** [comment]
Woo! Undeserved credit!
